extensions/wikia/WallNotifications/WallNotifications.class.php

```diff
@@ -1051,4 +1066,18 @@ protected function getUser( $userId ) {
 		return $this->cachedUsers[$userId];
 	}
 
+	private function getEmailExtensionController( $notification ) {
```

Comment:
Similar to `assertValidParams` inside of ForumController.class.php, what do you think about moving this method up to around L455? It's defined down here but called way higher up on L396.